### PR TITLE
V1.2 erwin

### PR DESCRIPTION
**Pull request by erwinqxy**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -1,18 +1,17 @@
 package seedu.address.logic.commands;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CLASSID;
```
Comment (AllardQuek) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:4]: Could consider splitting CLASSID into two words? i.e. `PREFIX_CLASS_ID`

Similarly in other locations and for STUDENTID as well.

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 2]
```diff
@@ -38,17 +32,16 @@
...
             + "Example: " + COMMAND_WORD + " 1 "
-            + PREFIX_PHONE + "91234567 "
-            + PREFIX_EMAIL + "johndoe@example.com";
+            + PREFIX_STUDENTID + "A0121234H "
+            + PREFIX_CLASSID + "B01";
 
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book.";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the ProgrammerError.";
```
Comment (AllardQuek) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:44]: Should we omit the 'the'? i.e. "This persona already exists in ProgrammerError."

File: src/main/java/seedu/address/model/person/ClassId.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    @Override
+    public String toString() {
+        return classId;
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        return other == this // short circuit if same object
+                || (other instanceof ClassId // instanceof handles nulls
+                && classId.equals(((ClassId) other).classId)); // state check
```
Comment (AllardQuek) [inline src/main/java/seedu/address/model/person/ClassId.java:50]: Not sure if this counts as a [complicated expression](https://nus-cs2103-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-complicated-expressions).

File: src/main/java/seedu/address/logic/commands/EditCommand.java (hunk 2 above)
Comment (erwinqxy) [reply src/main/java/seedu/address/logic/commands/EditCommand.java:44]: I think we can!

File: src/main/java/seedu/address/logic/commands/AddCommand.java (hunk 1 above)
Comment (erwinqxy) [reply src/main/java/seedu/address/logic/commands/AddCommand.java:4]: Noted. I will work on it!

File: src/main/java/seedu/address/logic/commands/EditCommand.java (hunk 2 above)
Comment (shermannws) [reply src/main/java/seedu/address/logic/commands/EditCommand.java:44]: I think allard suggestion is good!

File: src/main/java/seedu/address/model/person/ClassId.java (hunk 3 above)
Comment (AllardQuek) [reply src/main/java/seedu/address/model/person/ClassId.java:50]: Yup sure, not a priority for now.

File: src/main/java/seedu/address/logic/commands/AddCommand.java (hunk 1 above)
Comment (erwinqxy) [reply src/main/java/seedu/address/logic/commands/AddCommand.java:4]: Done with this!

**Review by AllardQuek — APPROVED** [review]

**Pull request merged**